File: Experimento.Domain/Entities/Ride.cs
namespace Experimento.Domain.Entities;

public class Ride
{
    public string Id { get; set; }
    public string VehicleId { get; set; }
    public virtual Vehicle Vehicle { get; set; }
    public string RiderId { get; set; }
    public virtual User Rider { get; set; }
    public DateTime Date { get; set; }
}

File: java/src/main/java/com/galileucopilot/galileucopilot/entities/Ride.java
package com.galileucopilot.galileucopilot.entities;


import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Date;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "table_ride")
public class Ride {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @ManyToOne
    @JoinColumn(name = "vehicle_id")
    private Vehicle vehicle;

    @ManyToOne
    @JoinColumn(name = "rider_id")
    private User rider;

    private Date date;
}